src/style-flight.scss

```diff
@@ -5,9 +5,11 @@
 
 // Import Auro tokens
 @import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import '../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
```

Comment:
Same comment here.

Reply:
This is for the util_displayHiddenVisually classes to function on the components.